### PR TITLE
FE-1119 - Incorporate UX feedback for the Sending Domains feature

### PR DESCRIPTION
**Pull request by nicklemmon**
[FE-1119](https://jira.int.messagesystems.com/browse/FE-1119)

### What Changed

- Update Sending Domains feature according to UX feedback

### How To Test

Go to `/account/sending-domains/edit/<domain>` and verify the following:

- [x] The "Verified" checkmark is now using the `<CheckCircle/>` icon which is placed to the right of the text
- [x] The "Verified" text should use the font size 300, font weight 400, and color gray 900 design tokens
- [x] The `<Check/>` icon should render inside the `Verify ___ Record` Panel actions to the right of the text content
- [x] The "Need help verifying your sending domain?" text on the lefthand side should match the color of similar text within that lefthand column
- [x] For an unverified domain, click on "set this domain up for sending via email" on the lefthand side and verify that opens now meets current design standards. There are two variants of the modal - modify lines 146-148 of `src/pages/sendingDomains/components/VerifyEmail.js` to see both variants
- [x] Verifying nothing looks broken in the OG theme

### To Do
- [ ] Incorporate feedback

### CODE REVIEW

File: src/pages/sendingDomains/components/SetupSending.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ export class SetupSending extends Component {
 
     if (!readyFor.sending && !readyFor.dkim) {
       content = (
-        <SubduedText>
+        <SubduedText mt="300">
```
Comment (bkemper) [inline src/pages/sendingDomains/components/SetupSending.js:59]: ⛰️

File: src/pages/sendingDomains/components/VerificationHelp.js [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
-  return <p>
-    Need help verifying your domain? <a href="https://www.sparkpost.com/docs/getting-started/getting-started-sparkpost/#sending-domain-step-2-verifying-domain-ownership" rel="noopener noreferrer" target="_blank">Follow this guide.</a>
-  </p>;
+  return (
+    <SubduedText mt="300">
+      <span>Need help verifying your domain? </span>
+      <a
+        href="https://www.sparkpost.com/docs/getting-started/getting-started-sparkpost/#sending-domain-step-2-verifying-domain-ownership"
+        rel="noopener noreferrer"
+        target="_blank"
```
Comment (bkemper) [inline src/pages/sendingDomains/components/VerificationHelp.js:15]: Shouldn't we use `<ExternalLink />`?

File: src/pages/sendingDomains/components/tests/SetupInstructionPanel.test.js [hunk 3]
```diff
@@ -47,9 +47,8 @@ describe('SetupInstructionPanel', () => {
     const wrapper = subject({ isVerified: true });
 
     it('renders a re-verify button', () => {
-      expect(wrapper).toHaveProp('actions', [
-        expect.objectContaining({ content: 'Re-verify CNAME Record' }),
-      ]);
+      // 😳 Very tightly coupled to the implementation - ideally we would just be looking at the DOM here
+      expect(wrapper.props().actions[0].content.props.children).toBe('Re-verify CNAME Record');
```
Comment (bkemper) [inline src/pages/sendingDomains/components/tests/SetupInstructionPanel.test.js:51]: 😆

File: src/pages/sendingDomains/components/tests/VerifyEmail.test.js [hunk 4]
```diff
@@ -42,15 +42,23 @@ describe('VerifyEmail component', () => {
   it('should send abuse-at email', () => {
     const verifyAbuse = jest.fn().mockResolvedValue({});
     const wrapper = subject({ verifyAbuse });
-    wrapper.find('Button').at(1).simulate('click');
+
+    wrapper
+      .find('VerifyButton')
+      .at(1)
+      .dive()
```
Comment (bkemper) [inline src/pages/sendingDomains/components/tests/VerifyEmail.test.js:49]: Is `dive` necessary?

File: src/pages/sendingDomains/components/VerificationHelp.js (hunk 2 above)
Comment (nicklemmon) [reply src/pages/sendingDomains/components/VerificationHelp.js:15]: Yep! Just Prettier reformatting, but this is as good a time as any to incorporate that.

File: src/pages/sendingDomains/components/tests/VerifyEmail.test.js (hunk 4 above)
Comment (nicklemmon) [reply src/pages/sendingDomains/components/tests/VerifyEmail.test.js:49]: Might not be. Let me check.

**Pull request merged**